src/Driver/Driver.php

```diff
@@ -37,13 +39,67 @@ interface Driver
      */
     public const LINE_NOT_EXECUTABLE = -2;
 
+    protected $detectDeadCode = false;
```

Comment:
Note that this now necessarily defaults to `false` rather than `true` because otherwise `detectingDeadCode()` would return `true` for PHPDBG and PCOV even though `canDetectDeadCode` would `return false`